packages/leavittbook/src/components/titanium-address-input-demo.ts

```diff
@@ -0,0 +1,107 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, query } from 'lit/decorators.js';
+import { h1, p } from '@leavittsoftware/titanium-styles';
+import StoryStyles from '../styles/story-styles';
+import '@leavittsoftware/titanium-button';
+import '@leavittsoftware/titanium-card';
+import '@leavittsoftware/titanium-address-input';
+import '@api-viewer/docs';
+import '../shared/code-block';
+import '../shared/story-header';
+import { TitaniumAddressInput } from '@leavittsoftware/titanium-address-input';
+
+@customElement('titanium-address-input-demo')
+export class TitaniumAddressInputDemoElement extends LitElement {
+  @query('titanium-address-input[id="1"]') private defaultAddress!: TitaniumAddressInput;
+  static styles = [
+    h1,
+    p,
+    StoryStyles,
+    css`
+      titanium-button {
+        padding-bottom: 16px;
+      }
+    `,
+  ];
+
+  async reset() {
+    this.defaultAddress.reset();
+  }
+
+  async layout() {
+    this.defaultAddress.layout();
+  }
+
+  #defaultStory() {
+    return html`
+      <div>
+        <h1>Default</h1>
+        <p>Default address input styles</p>
+        <titanium-button @click=${this.reset} raised>Reset input</titanium-button>
+        <titanium-button @click=${this.layout} raised>Layout input</titanium-button>
+        <titanium-address-input id="1" googleMapsApiKey="AIzaSyBO1C4Ek3L3sswvLxCjWIN-xgZayWyhp-k"></titanium-address-input>
+      </div>
+    `;
+  }
+
+  #requiredStory() {
+    return html`
+      <div>
+        <h1>Required</h1>
+        <p>Required address input styles</p>
+        <titanium-address-input
+          googleMapsApiKey="AIzaSyBO1C4Ek3L3sswvLxCjWIN-xgZayWyhp-k"
+          required
+          validation-message="Please input an address"
+        ></titanium-address-input>
+      </div>
+    `;
+  }
+
+  #helperStory() {
+    return html`
+      <div>
+        <h1>Helper</h1>
+        <p>Address input styles with helper text</p>
+        <titanium-address-input googleMapsApiKey="AIzaSyBO1C4Ek3L3sswvLxCjWIN-xgZayWyhp-k" helper="Address for something"></titanium-address-input>
+      </div>
+    `;
+  }
+
+  #helperPersistStory() {
+    return html`
+      <div>
+        <h1>Persistent helper</h1>
+        <p>Address input styles with persistent helper text</p>
+        <titanium-address-input
+          googleMapsApiKey="AIzaSyBO1C4Ek3L3sswvLxCjWIN-xgZayWyhp-k"
+          helper="Address for something"
+          helper-persistent
```

Comment:
This isn't declared as an attribute in the component so to make it work it needs to be `helperPersistent`